src/formats/dng/dng-output-color.ts

```diff
@@ -0,0 +1,19 @@
+export enum DngOutputColor
+{
+  // Raw color (unique to each camera).
+  Raw = 0,
+
+  // sRGB D65 (default).
+  SRGB = 1,
+
+  // Adobe RGB (1998) D65.
+  AdobeRGB = 2,
+
+  // Wide Gamut RGB D65.
+  WideGamutRGB = 3,
+
+  // Kodak ProPhoto RGB D65.
+  KodakProPhotoRGB = 4,
+
+  XYZ,
```

Comment:
Looks like I forgot to add `= 5` in Magick.NET. Could you add that here?

Reply:
Looks like a new value was added, just Added this to Magick.NET:

```
/// <summary>
/// Academy Color Encoding System.
/// </summary>
ACES = 6,
```

Reply:
I added `= 5` and new color